person.rb

```diff
@@ -1,14 +1,16 @@
-require_relative './corrector'
+require './corrector'
 
```

Comment:
Please add a method that serves as the 'belongs-to' side (a student belongs to a classroom) in the `Student` class. Moreover please add a getter method for `classroom`. You may see [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) example to more about the `belongs_to` relationship.